app/css/style.css

```diff
@@ -7,7 +7,7 @@ html, body, iframe {
 
 md-card[md-ink-ripple] {
   position: relative;
-  width:100%;
+  /*width:100%;*/
   /*overflow-y: auto;*/
```

Comment:
Hmm... if both of these are now commented out, do we need either of them? Could be just deleted altogether.